### DISCUSSION TITLE
How to open a window in floating mod

### DISCUSSION THREAD

**Discussion by CRAG666** [wezterm/wezterm#3640] [Q&A]
Since I can have a floating window, I recently switched from kitty to this terminal, but when trying to assign a size it is not assigned, which is not the case with kitty

**Comment by CRAG666** [comment]
https://user-images.githubusercontent.com/34254373/235283871-d8cf365c-9231-458c-a624-8e13f72f13cb.mp4

**Comment by yutkat** [comment]
https://github.com/wez/wezterm/issues/270

**Reply by CRAG666** [reply]
It is not what I need, it is that it respects the size assignments that my composer gives, like kitty does

**Comment by firedes** [comment]
It seems your compositor is Hyprland? On Hyprland, add a window rule like this: `windowrulev2 = float,class:^(xx)$`, and then start a class overridden terminal: `wezterm start --class xx`, this new terminal window will be floating on Hyprland. You can also use Hyprland's window rules to adjust this floating window's position and size.

**Reply by CRAG666** [reply]
@YinAqu  I use newm as composer and class is added as appid, but with kitty everything goes perfect but with wezterm it doesn't

**Reply by ciesling** [reply]
You are a genius! Neat workaround for dealing with wezterm while on hyprland :D
